daml-lf/spec/daml-lf-1.rst

```diff
@@ -515,13 +541,16 @@ The literals represent actual Daml-LF values:
    integer, i.e. it equals ``2⁶³``.  Similarly,``2019-13-28`` is not a
    valid ``LitDate`` because there are only 12 months in a year.
 
-Number-like literals (``LitNatTyp``, ``LitInt64``,
-``LitNumeric``,``LitDate``, ``LitTimestamp``) are ordered by natural
-ordering. Text-like literals (``LitText``, ``LitParty``, and
+Number-like literals (``LitNatTyp``, ``LitInt64``, ``LitNumeric``,
+``LitBigNumeric``, ``LitDate``, ``LitTimestamp``) are ordered by
+natural ordering. Text-like literals (``LitText``, ``LitParty``, and
 ``Contract ID``) are ordered lexicographically. Note that in the ASCII
 encoding, the character ``#`` comes before digits, meaning V0 Contract
 ID are ordered before V1 Contract ID. In the following we will denote
-the corresponding (non-strict) order by ``≤ₗ``.
+the corresponding (non-strict) order by ``≤ₗ``.  RoundingMode literals
+are ordered as they appear in the definition of ``LitRoundingMode``,
+e.g. ``ROUNDING_UP`` and ``ROUNDING_UNNECESSARY`` are the smallest and
+greatest rounding mode.
```

Comment:
Do we need ordering on rounding modes? I would have thought no

Reply:
Can’t we fail in the typechecker instead of adding a dynamic check?

Reply:
We cannot fail in the type checker, as comparison and equality are generic. 
This is precisely why comparison of functions type checks.

Reply:
oh I remember now makes sense.